### DISCUSSION TITLE
Failed to load jvm.dll

### DISCUSSION THREAD

**Discussion by TimZhang13579** [dbeaver/dbeaver#13850] [Q&A] [answered]
#### System information: 

Windows 10 Enterprise Edition.
DBeaver 7.3.5.
Java 1.8.0_301.

#### Describe the problem you're observing:

I Failed to launch DBeaver, it always shows me a dialog with error message: Failed to load the JNI shared library "C:\Program Files\Java\jre 1.8.0_301\bin\server\jvm.dll

![image](https://user-images.githubusercontent.com/74940838/131475538-13b3dd89-4aa4-447c-bd65-5f4b7979a1a8.png)

#### Steps to reproduce, if exist:

1. Install JDK and JRE.
2. Install DBeaver.
3. Click DBeaver shortcut to start the program.
4. An error message shows immediately.

**Comment by SubOptimal** [comment]
Might be a problem of mixed 32 / 64 bit executables / libraries.

Could you please post the output of `C:\Program Files\Java\jre 1.8.0_301\bin\java -version`

**Comment by TimZhang13579** [comment]
Hi @SubOptimal ,
Please have a look at the picture. Thanks.
![image](https://user-images.githubusercontent.com/74940838/131603889-c0a9f0a4-4eae-4991-b40f-59c9e2fe8ea7.png)

**Comment by ShadelessFox** [comment, marked as answer]
Hello @TimZhang13579,

DBeaver is bundled with JRE. The system-installed one can cause problems just like in this case.

Run the installer again and make sure the <kbd>Include Java</kbd> option is checked.
